eclipse-collections/src/main/java/org/eclipse/collections/impl/set/sorted/mutable/MutableSortedSetFactoryImpl.java

```diff
@@ -25,6 +25,12 @@ public <T> MutableSortedSet<T> empty()
         return TreeSortedSet.newSet();
     }
 
+    @Override
+    public <T> MutableSortedSet<T> empty(Comparator<? super T> comparator)
+    {
+        return TreeSortedSet.newSet(comparator);
```

Comment:
@Desislav-Petrov Please move the code from with(Comparator) to here and change with(Comparator) to call empty().

Reply:
Good spot @donraab  - addressed